apps/studio/components/interfaces/Auth/Hooks/CreateHookSheet.tsx

```diff
@@ -290,196 +290,199 @@ export const CreateHookSheet = ({
           </div>
         </SheetHeader>
         <Separator />
-        <Form_Shadcn_ {...form}>
-          <form
-            id={FORM_ID}
-            className="space-y-6 w-full py-8 flex-1"
-            onSubmit={form.handleSubmit(onSubmit)}
-          >
-            <FormField_Shadcn_
-              key="enabled"
-              control={form.control}
-              name="enabled"
-              render={({ field }) => (
-                <FormItemLayout
-                  layout="flex"
-                  className="px-8"
-                  label={`Enable ${values.hookType}`}
-                  description={
-                    values.hookType === 'Send SMS hook'
-                      ? 'SMS Provider settings will be disabled in favor of SMS hooks'
-                      : undefined
-                  }
-                >
-                  <FormControl_Shadcn_>
-                    <Switch
-                      checked={field.value}
-                      onCheckedChange={field.onChange}
-                      disabled={field.disabled}
-                    />
-                  </FormControl_Shadcn_>
-                </FormItemLayout>
-              )}
-            />
-            <Separator />
-            {httpsAuthHooksEnabled && (
+        <div className="overflow-auto flex-grow">
```

Comment:
looks like lots of changes here, but view without whitespace — only added this wrapping div

Reply:
Replace this div with `SheetSection`. You can view the changes without whitespace [here](https://github.com/supabase/supabase/pull/29746/files?diff=split&w=1)